### DISCUSSION TITLE
12/26 조세용 블로그 3회차

### DISCUSSION THREAD

**Discussion by devSeyong** [Orm-camp/blog-feedback#56] [동글동글 4기 피드백]
https://info6604.tistory.com/1

**Comment by Jungminchae** [comment]
- 데코레이터가 입문할 때 이해하기 어려운 내용이라 저번 기수에서도 데코레이터 질문을 많이 받았던 기억이 있는데 열심히 공부하시고 정리하신 느낌이나네요
- 블로그 스타일에 맞게 소제목들을 좀 더 큰 요소로 사용해주시면 좋을 것 같습니다.
